### DISCUSSION TITLE
`hx --grammar fetch` fails

### DISCUSSION THREAD

**Discussion by Zoybean** [helix-editor/helix#7802] [Troubleshooting] [answered]
### Summary

One grammar fails to fetch, after deleting grammars.

### Reproduction Steps

 

I tried this:
1. `rm -r runtime/grammars`
2. `hx --grammar fetch`

I expected this to happen:
- grammars update successfully

Instead, this happened:
```
Fetching 153 grammars
152 updated grammars
        astro              now on 5f5c3e73c45967df9aa42f861fad2d77cd4e0900
...(147 lines omitted)...
        zig                now on 8d3224c3bd0890fe08358886ebf54fca2ed448a6
Failure 1/1: hare Git command failed.
Stdout:
Stderr: error: Server does not allow request for unadvertised object bc26a6a949f2e0d98b7bfc437d459b250900a165

Error: 1 grammars failed to fetch
```

### Helix log

No relevant log entries

### Platform

Windows

### Terminal Emulator

Windows Terminal Version: 1.17.11461.0

### Helix Version

23.05-280-g99413558

**Comment by pascalkuthe** [comment]
your git version islkely out of date, one of the grammar (for hare) is hosted on sourcehut which requires a somewhat recent git version

**Reply by pascalkuthe** [reply, marked as answer]
see https://github.com/helix-editor/helix/issues/7036#issuecomment-1556265085
